### ISSUE TITLE
Cannot type into search field after first start of KeepassXC

### ISSUE THREAD

**Issue by imp1sh** [keepassxreboot/keepassxc#5608]
When I start KeepassXC and unlock the database I cannot search in it. I hit CTRL + F or I click with mouse into search field. Then I cannot enter any string whatsoever. When I close Keepass and restart / unlock it again it works again. But the first time after system boot it always fails to work

## Steps to Reproduce

- Install KeepassXC

- Start Keepass

- Try to search for an element via CTRL + F or by clicking into the search form

## Expected Behavior
I should be able to search but I'm not.

## Context
I experienced this phenomena on two different computers, both installed freshly with Fedora 33.

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.6.1
Revision: 9a35bba

Qt 5.15.1
Diagnosemodus ist deaktiviert.

Betriebssystem: Fedora 33 (Workstation Edition)
CPU-Architektur: x86_64
Kernel: linux 5.8.16-300.fc33.x86_64

Aktivierte Erweiterungen:
- Auto-Type
- Browser-Integration
- SSH-Agent
- KeeShare (bestätigtes und unbestätigtes Teilen)
- YubiKey
- Secret-Service-Integration

Kryptographische Bibliotheken:
 libgcrypt 1.8.6

[NOTE]: # ( Pick choices based on your environment )
Operating System: Fedora 33
Desktop Env: Gnome
Windowing System: Wayland

**Comment by droidmonkey** [comment]
Screenshot when it's not working please

**Comment by imp1sh** [comment]
Here you go.
![keepass_not_search_possible](https://user-images.githubusercontent.com/3486156/97088529-ba50d400-1631-11eb-926f-634f6ab1e5ac.png)

**Comment by imp1sh** [comment]
Here you go.
![keepass_not_search_possible](https://user-images.githubusercontent.com/3486156/97088655-ae194680-1632-11eb-882e-13a849f872d5.png)

**Comment by droidmonkey** [comment]
Seems like the field is enabled and focused. You cannot type into it at all? This is definitely not reproducible on any of my machines.

**Comment by imp1sh** [comment]
No, I cannot type at all, no matter if I klick into the field nor when I hit CTRL + F.
It must have something to do with changes with Fedora 33. KeepassXC act strangely.

**Comment by rodmcrae** [comment]
I'm also having trouble with search, on Debian 10 (stable). I can type into the search field, but nothing comes up - totally disfunctional search. I'm sure this was working before, though I haven't used Keepassxc before, only Keepassx. Strangely, I have the same issue in Keepassx (I have both installed).

**Comment by droidmonkey** [comment]
Nothing was changed with search in 2.6.2 that would cause this behavior.

**Comment by stinos** [comment]
Related (perhaps): when hiding the toolbar or even changing the window size so that the search field is not visible, search functionality is disabled, i.e. Ctr-lF does nothing.
As a result if you change settings to have the toolbar show both icons and text, search is possible only when making the window wide enough. Note that when the window is large enough and you start typing in the search field, but then shrink the window to make the field disappear, the search field is even though hidden still active because typing text does alter search results, until pressing Esc.

**Comment by phoerious** [comment]
That is somewhat of a Qt bug and we may need to develop a workaround for it.

**Comment by stinos** [comment]
I assume Ctrl-F is implemented as 'move keyboard focus to search textbox' somewhere, and that doesn't work if textbox isn't visible?
Niceset fix would probably be something along the lines of displaying a popup/overlay with the search box upon Ctrl-F.

**Comment by droidmonkey** [comment]
That issue is tracked elsewhere

**Comment by stinos** [comment]
Sorry didn't find this at first; here it is though: #505

**Comment by imp1sh** [comment]
If can do anything to resolve this issue just tell me.

**Comment by finderAUT** [comment]
I am experiencing the same issue on Fedora 33 but it is not just limited to the search input field but also the ones of e.g. add entry

KeePassXC - Version 2.6.2
Revision: e9b9582

Qt 5.15.1
Diagnosemodus ist deaktiviert.

Betriebssystem: Fedora 33 (Workstation Edition)
CPU-Architektur: x86_64
Kernel: linux 5.8.16-300.fc33.x86_64

Aktivierte Erweiterungen:
- Auto-Type
- Browser-Integration
- SSH-Agent
- KeeShare (bestätigtes und unbestätigtes Teilen)
- YubiKey
- Secret-Service-Integration

Kryptographische Bibliotheken:
 libgcrypt 1.8.6

**Comment by droidmonkey** [comment]
The only common denominator amongst all you folks (besides KeePassXC) is Fedora 33.

**Comment by droidmonkey** [comment]
I loaded a Fedora 33 instance with XFCE and could not reproduce this problem.

**Comment by imp1sh** [comment]
I use Gnome with Wayland. Maybe Wayland is the reason idk. I have couple of other different issues actually. Sometimes it's not possible to paste a password into keepassxc. Also the ALT + TAB issue for which I made a ticket.
Don't know if they are related though.

**Comment by finderAUT** [comment]
> I loaded a Fedora 33 instance with XFCE and could not reproduce this problem.

Probably is a Gnome/Wayland issue .... it works with Xorg.

**Comment by droidmonkey** [comment]
@imp1sh if you are having all those issues it is definitely a Wayland issue, perhaps even a qt5-wayland platform issue.

**Comment by imp1sh** [comment]
Where to put this then? I have not experienced any issues with other applications so far, only keepassxc.
@finderAUT 
Xorg is no longer being maintained.

**Comment by droidmonkey** [comment]
Are you running kpxc with wayland or using xwayland? QT_QPA_PLATFORM should be set to wayland.

**Comment by imp1sh** [comment]
I'm don't know how to check this? I haven't made any manual changes to the base system of Fedora 33.

**Comment by scandinave** [comment]
start KPXC with the option -platform wayland seem to resolve the problem.

**Comment by agittins** [comment]
I'm experiencing the same symptoms as @finderAUT, in that no input fields respond to keyboard input, with the exception of the password fields (but only when they are in hidden-mode and even then, intermittently). Widgets do receive focus (they get highlights around them) on mouse actions, and context menus work (including pasting text) but they don't respond to keyboard actions, including Ctrl-V or TAB.

Per @imp1sh 's experience, if I close and re-start kpxc, things appear to work OK - although I get no caret showing in single-line text inputs, but typing works (the "notes" multiline widget seems to work fine - caret and input ok).

I'm using Debian sid and i3wm. I think this started happening after I did a system upgrade (and it was a big one that included qt and a million other things!) but I don't recall if it was the same time that I updated kpxc or not from 2.5ish - it also coincides with moving to a new machine, so I'm clearly not optimising for reduced variables over here! :-)

I have kpxc starting automatically from my i3 config, so it could be a race condition, which might explain why the second launch works. picom (my compositor) launches before it, but maybe it's still setting up and causes an issue. I'm running Xorg.

I haven't noticed issues in any other Qt apps, and in fact the caret issue doesn't seem to be a problem in them, either. I've tried qjackctl, qmidiroute and vlc and text input works fine on those.

```
KeePassXC - Version 2.6.2
Revision: e9b9582

Qt 5.15.1
Debugging mode is disabled.

Operating system: Debian GNU/Linux bullseye/sid
CPU architecture: x86_64
Kernel: linux 5.9.0-4-amd64

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (only unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
 libgcrypt 1.8.7
```

If there's more I can do to help narrow down the culprit let me know.

**Comment by muety** [comment]
Got the same problem on Fedora 33 and latest Gnome.

**Comment by fazuuuu** [comment]
Same issue on fedora 33, using gnome on wayland.

**Comment by rtoepfer** [comment]
I'm having the same issue on Fedora 33.  I've done a full dnf install update.  At times I'm unable to enter via the keyboard.  Its not always on initial application start.  The bug also occurs after switching applications.  To fix I don't have to close the entire application but just close the database and then reopen the database.  I'm using kdbx database format.

KeePassXC - Version 2.6.4
Revision: 34a78f0

Qt 5.15.2
Debugging mode is disabled.

Operating system: Fedora 33 (Workstation Edition)
CPU architecture: x86_64
Kernel: linux 5.10.16-200.fc33.x86_64

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- libgcrypt 1.8.7

**Comment by imp1sh** [comment]
What is the next step? Do you need any more input from us users?
I think the bug is easy to reproduce in contrast to the issues flag. I myself has witnessed this behaviour on every single installation which are about four so far.

**Comment by droidmonkey** [comment]
I just installed Fedora 33 in a VM, using GNOME and Wayland, installed KeePassXC from the store, created a new database, unlocked.... and no problems searching. Enabled start on login, logged out and back in, unlocked DB and no problems searching.

**Comment by imp1sh** [comment]
That's weird. It happens, I can promise you that. What can I do to narrow it down? Strace? Debug Symbols?
I don't know what to do. Is this maybe an issue for some other party?

**Comment by droidmonkey** [comment]
File a bug report with qt under the qt5wayland module: https://bugreports.qt.io
I won't do it because I cannot reproduce the problem.

**Comment by auzias** [comment]
Hi, same issue for me. On Fedora 33 with Wayland too.

The ugly work-around I found is to use the mouse in the search field on the loop icon and two-click on enable/disable `Case sensitive`. It works sometimes with just one click.

```
KeePassXC - Version 2.6.4
Revision: 34a78f0

Qt 5.15.2
Debugging mode is disabled.

Operating system: Fedora 33 (Workstation Edition)
CPU architecture: x86_64
Kernel: linux 5.10.20-200.fc33.x86_64

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- libgcrypt 1.8.7
```

**Comment by droidmonkey** [comment]
Can someone post a screen capture video of the behavior?

**Comment by droidmonkey** [comment]
Can someone confirm if they have anything related to "deepin" installed on their systems? Apparently `deepin-qt5-integration` causes significant issues with every single Qt app if it is installed.

See writeup for a similar issue plaguing a Manjaro user:
https://github.com/keepassxreboot/keepassxc/issues/6277#issuecomment-801436082

**Comment by droidmonkey** [comment]
I just confirmed that installing any of these packages (then restarting) causes the input fields of KeePassXC to be almost unusable:

![image](https://user-images.githubusercontent.com/2809491/111543881-0bc93f00-874a-11eb-9730-bf2b86ef7513.png)

**Comment by imp1sh** [comment]
[jochen@localhost ~]$ rpm -aq|egrep -i 'deepin|qt5'|sort
adwaita-qt5-1.2.0-1.fc33.x86_64
dbusmenu-qt5-0.9.3-0.25.20160218.fc33.x86_64
libaccounts-qt5-1.16-2.fc33.x86_64
libadwaita-qt5-1.2.0-1.fc33.x86_64
phonon-qt5-4.11.1-6.fc33.x86_64
phonon-qt5-backend-gstreamer-4.10.0-4.fc33.x86_64
polkit-qt5-1-0.113.0-5.fc33.x86_64
poppler-qt5-0.90.0-6.fc33.x86_64
qt5-qtbase-5.15.2-2.fc33.x86_64
qt5-qtbase-common-5.15.2-2.fc33.noarch
qt5-qtbase-gui-5.15.2-2.fc33.x86_64
qt5-qtdeclarative-5.15.2-2.fc33.x86_64
qt5-qtgraphicaleffects-5.15.2-2.fc33.x86_64
qt5-qtlocation-5.15.2-2.fc33.x86_64
qt5-qtmultimedia-5.15.2-2.fc33.x86_64
qt5-qtquickcontrols2-5.15.2-2.fc33.x86_64
qt5-qtquickcontrols-5.15.2-2.fc33.x86_64
qt5-qtsensors-5.15.2-2.fc33.x86_64
qt5-qtspeech-5.15.2-2.fc33.x86_64
qt5-qtspeech-speechd-5.15.2-2.fc33.x86_64
qt5-qtsvg-5.15.2-2.fc33.x86_64
qt5-qttools-common-5.15.2-2.fc33.noarch
qt5-qttools-libs-designer-5.15.2-2.fc33.x86_64
qt5-qtwayland-5.15.2-3.fc33.x86_64
qt5-qtwebchannel-5.15.2-2.fc33.x86_64
qt5-qtwebkit-5.212.0-0.53.alpha4.fc33.x86_64
qt5-qtx11extras-5.15.2-2.fc33.x86_64
qt5-qtxmlpatterns-5.15.2-2.fc33.x86_64
qt5-srpm-macros-5.15.2-1.fc33.noarch
qtlockedfile-qt5-2.4-33.20150629git5a07df5.fc33.x86_64
qtsingleapplication-qt5-2.6.1-40.fc33.x86_64
quazip-qt5-0.7.6-9.fc33.x86_64

**Comment by auzias** [comment]
I remove the `deepin`'s, it works like a charm so far.

**Thank you !**

@droidmonkey, by curiosity, could you tell us how you find it ?

**Comment by droidmonkey** [comment]
Wasn't me! It was a fellow community member. See my comment about this a couple spots up.

**Comment by finderAUT** [comment]
Like @imp1sh I don't have any deepin packages installed.
Did someone already file a QT bug?

I also tried to reproduce it in a VM but so far didn't succeed.

**Comment by droidmonkey** [comment]
These packages are the source of the problem for @imp1sh. I installed qtsingleapplication-qt5, rebooted, and the problem appeared immediately. qtlockedfile-qt5 comes along with it and should also be uninstalled.

```
qtlockedfile-qt5-2.4-33.20150629git5a07df5.fc33.x86_64
qtsingleapplication-qt5-2.6.1-40.fc33.x86_64
```

**Comment by Germano0** [comment]
@droidmonkey there are no such dependencies in keepassxc
https://src.fedoraproject.org/rpms/keepassxc/blob/rawhide/f/keepassxc.spec

**Comment by droidmonkey** [comment]
This is not a dependency in KeePassXC, this is a package that causes negative behavior in KeePassXC because it modifies the Qt5 environment.

**Comment by imp1sh** [comment]
Thank you so much for the hint @droidmonkey. I have just deinstalled those packages and I will report if they solve the problem.
It looks like those were packages were installed as a dependency for smplayer and mpv.

**Comment by Germano0** [comment]
@droidmonkey in order to start a discussion in Fedora development community to ask for removal of those two Qt libraries, I need detailed infos about. Where is stated in official Qt docs, that they are deprecated?
Thank you

**Comment by rtoepfer** [comment]
I do not have deepin*, qtlocked or qtsingleapp packages on my default
Fedora 33 install and I am still having issues.

Its been a few years since I've been working fully on Linux.  Is there a
static linked KeePassXC available?  What about another way of packaging the
dependencies along with KeePassXC itself?

**Comment by droidmonkey** [comment]
@Germano0 I may be a bit premature to request removal, but something with Fedora, Wayland, and these qt5 libraries is causing major problems.

@rtoepfer you'll have to figure out perhaps yet another package that is causing similar interference.

**Comment by muety** [comment]
I've been following this discussion for a while. I'd like to report that I have neither of those Qt packages installed, but the issue is still occurring on my system.

**Comment by ladislavfurman** [comment]
can confirm - this happens on fedora 33 not only after first boot, but after db is locked when minimized and opened after that.

**Comment by droidmonkey** [comment]
I did some more debugging and found that this only happens when KeePassXC starts on login. Once I quit and reopen KeePassXC then the issue is not present, even with the above applications installed.

**Comment by finderAUT** [comment]
I can confirm that for me it only happens when KeePassXC autostarts on login. If I (re)start it manually it's fine.

Removing
`qtlockedfile-qt5-2.4-33.20150629git5a07df5.fc33.x86_64`              
`qtsingleapplication-qt5-2.6.1-40.fc33.x86_64`
which came as dependency of texstudio didn't help.
Don't know why but there is a diff in the naming "qtsingleapplication-qt5 **-0**"
```
$ dnf repoquery --whatdepends qtlockedfile-qt5-2.4-33.20150629git5a07df5.fc33.x86_64  
qtlockedfile-qt5-devel-0:2.4-33.20150629git5a07df5.fc33.i686
qtlockedfile-qt5-devel-0:2.4-33.20150629git5a07df5.fc33.x86_64
qtsingleapplication-qt5-0:2.6.1-40.fc33.x86_64
qtsinglecoreapplication-qt5-0:2.6.1-40.fc33.x86_64
vacuum-im-0:1.3.0-0.20.20200608gitb6c5dad.fc33.x86_64

$  dnf repoquery --whatdepends qtsingleapplication-qt5-0:2.6.1-40.fc33.x86_64 --userinstalled
texstudio-0:3.1.1-1.fc33.x86_64
```

Other qt5 packages I have installed which differ from a fresh Fedora 33 install + dnf upgrade:
```
poppler-qt5-0.90.0-6.fc33.x86_64
python3-pyqt5-sip-4.19.24-1.fc33.x86_64
python3-qt5-5.15.0-4.fc33.x86_64
python3-qt5-base-5.15.0-4.fc33.x86_64
python-qt5-rpm-macros-5.15.0-4.fc33.noarch
qt5-qtconnectivity-5.15.2-2.fc33.x86_64
qt5-qtlocation-5.15.2-2.fc33.x86_64
qt5-qtmultimedia-5.15.2-2.fc33.x86_64
qt5-qtscript-5.15.2-2.fc33.x86_64
qt5-qtsensors-5.15.2-2.fc33.x86_64
qt5-qtserialport-5.15.2-2.fc33.x86_64
qt5-qttools-common-5.15.2-2.fc33.noarch
qt5-qttools-libs-designer-5.15.2-2.fc33.x86_64
qt5-qttools-libs-help-5.15.2-2.fc33.x86_64
qt5-qtwebchannel-5.15.2-2.fc33.x86_64
qt5-qtwebsockets-5.15.2-2.fc33.x86_64
qt5-srpm-macros-5.15.2-1.fc33.noarch
qtlockedfile-qt5-2.4-33.20150629git5a07df5.fc33.x86_64
qtsingleapplication-qt5-2.6.1-40.fc33.x86_64
```

**Comment by droidmonkey** [comment]
Those packages might be totally unrelated to the problem, I may have not been testing correctly given the autostart finding.

**Comment by LenzGr** [comment]
Just stumbled over this thread after experiencing the same problem (Fedora 33, x86_64, running keepassxc-2.6.4-1.fc33.x86_64 on Gnome/Wayland). I can confirm that text input does not work when KeepassXC is started automatically when logging in. When I quit the application and start it via the application menu (Application -> Accessories -> KeepassXC), text input works as expected after unlocking the database.

FWIW, this is how `~/.config/autostart/org.keepassxc.KeePassXC.desktop` looks like:
```
[Desktop Entry]
Name=KeePassXC
GenericName=Password Manager
Exec=/usr/bin/keepassxc
TryExec=/usr/bin/keepassxc
Icon=keepassxc
StartupWMClass=keepassxc
StartupNotify=true
Terminal=false
Type=Application
Version=1.0
Categories=Utility;Security;Qt;
MimeType=application/x-keepass2;
X-GNOME-Autostart-enabled=true
X-GNOME-Autostart-Delay=2
X-KDE-autostart-after=panel
```

And that's the desktop entry `/usr/share/applications/org.keepassxc.KeePassXC.desktop` for the start menu:
```
[Desktop Entry]
Name=KeePassXC
GenericName=Password Manager
GenericName[ar]=مدير كلمات المرور
GenericName[bg]=Мениджър на пароли
GenericName[ca]=Gestor de contrasenyes
GenericName[cs]=Aplikace pro správu hesel
GenericName[da]=Adgangskodehåndtering
GenericName[de]=Passwortverwaltung
GenericName[es]=Gestor de contraseñas
GenericName[et]=Paroolihaldur
GenericName[fi]=Salasanamanageri
GenericName[fr]=Gestionnaire de mot de passe
GenericName[hu]=Jelszókezelő
GenericName[id]=Pengelola Sandi
GenericName[it]=Gestione password
GenericName[ja]=パスワードマネージャー
GenericName[ko]=암호 관리자
GenericName[lt]=Slaptažodžių tvarkytuvė
GenericName[nb]=Passordhåndterer
GenericName[nl]=Wachtwoordbeheer
GenericName[pl]=Menedżer haseł
GenericName[pt_BR]=Gerenciador de Senhas
GenericName[pt]=Gestor de palavras-passe
GenericName[ro]=Manager de parole
GenericName[ru]=менеджер паролей
GenericName[sk]=Správca hesiel
GenericName[sv]=Lösenordshanterare
GenericName[th]=แอพจัดการรหัสผ่าน
GenericName[tr]=Parola yöneticisi
GenericName[uk]=Розпорядник паролів
GenericName[zh_CN]=密码管理器
GenericName[zh_TW]=密碼管理員
Comment=Community-driven port of the Windows application “KeePass Password Safe”
Comment[da]=Fællesskabsdrevet port af Windows-programmet “KeePass Password Safe”
Comment[et]=Kogukonna arendatav port Windowsi programmist KeePass Password Safe
Exec=keepassxc %f
TryExec=keepassxc
Icon=keepassxc
StartupWMClass=keepassxc
StartupNotify=true
Terminal=false
Type=Application
Version=1.0
Categories=Utility;Security;Qt;
MimeType=application/x-keepass2;application/x-keepassxc;
X-Desktop-File-Install-Version=0.26
```

**Comment by Germano0** [comment]
This message clarifies once and for all the problems with Fedora/CentOS/RHEL   https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/3BVLBS4B3XHJEXFVGD7RK2ZMXZG6JQZT/
I am releasing soon a patch in the Fedora package

**Comment by Germano0** [comment]
@imp1sh 

1. What is your feedback about keepassxc-2.6.4-2 in Fedora?
2. Could you please send me an email to my `myname.mysurname@gmail.com` so I can save your e-mail address? I need a list of people that experienced problems with Wayland, since upstream Qt developers say to have fixed the Wayland problem on 5.15.2 but my empirical research says the opposite. So I will probably need your help in future for running some tests.

Thank you

**Comment by TomaszWaszczyk** [comment]
Same on Linux Mint, below more information:

```
System:    Kernel: 5.8.0-50-generic x86_64 bits: 64 compiler: N/A Desktop: Cinnamon 4.8.6 
           wm: muffin dm: LightDM Distro: Linux Mint 20.1 Ulyssa base: Ubuntu 20.04 focal 
Machine:   Type: Desktop Mobo: MSI model: H97M-G43(MS-7924) v: 1.0 serial: <filter> 
           UEFI: American Megatrends v: 2.4 date: 07/22/2014 
CPU:       Topology: Quad Core model: Intel Core i5-4460 bits: 64 type: MCP arch: Haswell rev: 3 
           L2 cache: 6144 KiB 
           flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx bogomips: 25598 
           Speed: 1378 MHz min/max: 800/3400 MHz Core speeds (MHz): 1: 1591 2: 1492 3: 1622 
           4: 1398 
Graphics:  Device-1: NVIDIA GP104 [GeForce GTX 1070] vendor: ASUSTeK driver: nvidia v: 460.73.01 
           bus ID: 01:00.0 chip ID: 10de:1b81 
           Display: x11 server: X.Org 1.20.9 driver: nvidia 
           unloaded: fbdev,modesetting,nouveau,vesa resolution: 1920x1080~60Hz, 1920x1080~60Hz 
           OpenGL: renderer: GeForce GTX 1070/PCIe/SSE2 v: 4.6.0 NVIDIA 460.73.01 
           direct render: Yes 
```

**Comment by BessieTheCookie** [comment]
I was previously experiencing this issue on Fedora and it seems to be fixed in 2.6.4-2.fc34. Auto-type also wasn't working previously and now everything works.

**Comment by droidmonkey** [comment]
It was a bad packaging patch that caused the problem in fc33

**Comment by imp1sh** [comment]
I haven't had any problems for a couple of weeks now. I'm on fc34 and keepassxc 2.6.6.
From my perspective this is resolved.

**Comment by agittins** [comment]
I'm no longer seeing the exact issue that was reported either (fields not accepting input). I'm on debian sid, so my case was probably a bit different from the fedora/wayland flavour of the issue. I'm still not getting a carat in the search field but that's a minor issue which doesn't affect usability, is probably something specific to my system and seems unrelated to this bugreport.

**Comment by Germano0** [comment]
> It was a bad packaging patch that caused the problem in fc33

???????????

**Comment by droidmonkey** [comment]
@Germano0 this was the Qt QPA Wayland patch that was being injected into the Fedora release.

**Comment by Altonss** [comment]
It looks like the issue has reappeard with the latest version (see #7959 )

**Comment by rvernica** [comment]
@droidmonkey I'm running into this issue pretty consistently on `2.7.1` on Fedora 35

**Comment by droidmonkey** [comment]
The common denominator is KeePassXC, possibly Wayland, and Fedora. The last time this cropped up it was external packages causing invalid Qt operation that impacted our app.

**Comment by ghost** [comment]
Occurs on Fedora 35 (Wayland) after system restarts. The database password field takes input. 
v2.7.1

**Comment by auzias** [comment]
After the last update, it does not work anymore for me neither.

Being on Fedora I rely on flatpak for several applications. **The search-field for [KeepassXC as a flatpak](https://www.flathub.org/apps/details/org.keepassxc.KeePassXC) works**.

Also, the issue described on this ticket is a bit different though since the search does not work at all (and not only on start).

If it helps, here is my `qt` packages :

```bash
# dnf list installed qt*
Installed Packages
qt.x86_64                                                  1:4.8.7-61.fc34                                   @updates 
qt-common.noarch                                           1:4.8.7-61.fc34                                   @updates 
qt-settings.noarch                                         34.7-2.fc34                                       @updates 
qt-x11.x86_64                                              1:4.8.7-61.fc34                                   @updates 
qt5-qtbase.x86_64                                          5.15.2-30.fc34                                    @updates 
qt5-qtbase-common.noarch                                   5.15.2-30.fc34                                    @updates 
qt5-qtbase-gui.x86_64                                      5.15.2-30.fc34                                    @updates 
qt5-qtdeclarative.x86_64                                   5.15.2-7.fc34                                     @updates 
qt5-qtsvg.x86_64                                           5.15.2-4.fc34                                     @fedora  
qt5-qtwayland.x86_64                                       5.15.2-19.fc34                                    @updates 
qt5-qtx11extras.x86_64                                     5.15.2-3.fc34                                     @anaconda
qt5-qtxmlpatterns.x86_64                                   5.15.2-4.fc34                                     @anaconda
```

**Comment by droidmonkey** [comment]
The Fedora package maintainer admitted it is a packaging issue and will be fixed soon.

**Comment by Germano0** [comment]
Please read 
https://github.com/keepassxreboot/keepassxc/issues/7959#issuecomment-1113550638
